test/karma.conf.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env node */
+
+var bourbonIncludePaths = require('node-bourbon').includePaths;
+
+module.exports = function(config) {
+  config.set({
+    basePath: '../',
+    frameworks: [
+      'jasmine'
+    ],
+    files: [
+      'test/specs/**/*.js'
+    ],
+    preprocessors: {
+      'test/specs/**/*.js': ['webpack', 'sourcemap']
+    },
+    webpack: {
+      module: {
+        loaders: [{
```

Comment:
Will also need to add `enforce: 'post'` or `enforce: 'pre'` for every loader object in here right?